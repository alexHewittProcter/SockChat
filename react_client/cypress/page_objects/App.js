export default class App {
  static base_url = 'http://localhost:3000/';
}
