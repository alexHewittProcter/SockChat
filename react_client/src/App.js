import React from 'react';
import './App.css';
import 'bootstrap/dist/css/bootstrap.min.css';
import MessageContainerComponent from './components/MessageContainer/MessageContainer';

class App extends React.Component {
  render() {
    return (
      <div className="App">
        <MessageContainerComponent />
      </div>
    );
  }
}

export default App;
