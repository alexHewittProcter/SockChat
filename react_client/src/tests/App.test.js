import React from 'react';
import App from '../App';
import { mount } from 'enzyme';

describe('App', () => {
  let app;
  beforeEach(() => {
    app = mount(<App />);
  });
  afterEach(() => {
    app.unmount();
  });
  it('Should render', () => {
    expect(app).toMatchSnapshot();
  });
});
