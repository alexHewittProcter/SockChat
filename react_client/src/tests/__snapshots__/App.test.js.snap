// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Should render 1`] = `
<div
  className="App"
>
  <div
    className="message-container"
  >
    <div
      className="message-list-wrapper"
    >
      <ul
        id="message-list"
      />
    </div>
    <div
      className="message-input-bar"
    >
      <InputGroup>
        <div
          className="input-group"
        >
          <FormControl
            name="message"
            onChange={[Function]}
            onKeyDown={[Function]}
            placeholder="Message..."
            value=""
          >
            <input
              className="form-control"
              name="message"
              onChange={[Function]}
              onKeyDown={[Function]}
              placeholder="Message..."
              value=""
            />
          </FormControl>
          <InputGroupAppend>
            <div
              className="input-group-append"
            >
              <Button
                active={false}
                disabled={false}
                onClick={[Function]}
                type="button"
                variant="primary"
              >
                <button
                  className="btn btn-primary"
                  disabled={false}
                  onClick={[Function]}
                  type="button"
                >
                  Send
                </button>
              </Button>
            </div>
          </InputGroupAppend>
        </div>
      </InputGroup>
    </div>
  </div>
</div>
`;
